### DISCUSSION TITLE
creating gguf model from lora adapter

### DISCUSSION THREAD

**Discussion by ragesh2000** [ggml-org/llama.cpp#5360] [Q&A] [answered]
I have a ggml adapter model created by convert-lora-to-ggml.py (ggml-adapter-model.bin).  Now my doubt is how to create the complete gguf model out of these? 
I have seen using 
`./main -m models/llama-13b.Q8_0.gguf --lora models/loras/adapter_model.bin --color -c 4096 --temp 0.7 --repeat_penalty >1.1 -n 256 -p "The conversation between human and AI assistant.\n[|Human|] Qual'è il significato della vita?\n[|AI|] " `
This is working fine for me. But this is just for querying right?
I want the merged full gguf model. How to create it ?

**Comment by BarfingLemurs** [comment, marked as answer]
use the export-lora binary to merge these:

`./export-lora -m model.gguf -l Lora -o merged-model.gguf`

**Comment by ragesh2000** [comment]
Thanks @BarfingLemurs
